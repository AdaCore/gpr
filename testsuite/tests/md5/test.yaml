description:
    Check MD5 checksum
driver: build_and_run
project_file: md5
main: main
