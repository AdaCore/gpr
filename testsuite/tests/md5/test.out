MD5: d6c0c878e4eee62a849c653aa8b80f0f
